src/server/services/validation-rules.js

```diff
@@ -25,24 +52,37 @@ function generateRules () {
   // for any values we format, we should format them the same way when we export
   for (const ruleType of Object.keys(rules)) {
     for (const rule of Object.values(rules[ruleType])) {
-      rule.valueFormatters = []
+      // validationFormatters are only applied when validating the records, so they
+      // aren't used during exports.
+      // persistentFormatters are always applied as soon as a value is read from a
+      // an upload, so they will affect both validation AND the exported value.
+      rule.validationFormatters = []
+      rule.persistentFormatters = []
 
       if (rule.dataType === 'String') {
-        rule.valueFormatters.push(recordValueFormatters.makeString)
-        rule.valueFormatters.push(recordValueFormatters.trimWhitespace)
+        rule.validationFormatters.push(recordValueFormatters.makeString)
+        rule.validationFormatters.push(recordValueFormatters.trimWhitespace)
```

Comment:
This would fix the "whitespace appearing in email addresses" issue that just came up

```suggestion
        rule.persistentFormatters.push(recordValueFormatters.trimWhitespace)
```